Util/PasswordUpdater.php

```diff
@@ -34,7 +34,7 @@ public function hashPassword(UserInterface $user)
     {
         $plainPassword = $user->getPlainPassword();
 
-        if (0 === strlen($plainPassword)) {
+        if ('' === $plainPassword) {
```

Comment:
This does not do the same when `$plainPassword` is `null` (i.e. you haven't set a new plain password)

Reply:
But `$plainPassword` should not be `null` 
https://github.com/FriendsOfSymfony/FOSUserBundle/blob/master/Model/UserInterface.php#L100

Reply:
it is definitely nullable (and it has always been). That's a mistake in the phpdoc.